docs/user-manual/en/persistence.md

```diff
@@ -456,6 +456,7 @@ To configure Apache ActiveMQ Artemis to use a database for persisting messages a
 
     The JDBC network connection timeout in milliseconds. The default value
     is 20000 milliseconds (ie 20 seconds).
+    When using a shared store it is recommended to set it less then or equal to `jdbc-lock-renew-period`.
```

Comment:
it should jdbc-lock-expiration instead of jdbc-lock-renew-period

Reply:
Oh I have used the old notes about it: good point!